components/look-at/index.js

```diff
@@ -39,6 +39,7 @@ AFRAME.registerComponent('look-at', {
   init: function () {
     this.target3D = null;
     this.vector = new THREE.Vector3();
+    this.isCamera = this.el.getObject3D('camera') ? true : false;
```

Comment:
Perhaps this should be called in `tick` in case the camera is not initialized in time.

`!!this.el.getObject3D('camera')`